src/callstack.cr

```diff
@@ -17,6 +17,19 @@ require "callstack/lib_unwind"
   require "debug/dwarf"
 {% end %}
 
+# `caller` returns the current execution stack (a backtrace) in the form
```

Comment:
Also, the documentation should start with a summary in 1-2 lines on the method role, then add details..
Example: [`BitArray#[]` documentation](https://github.com/crystal-lang/crystal/blob/3c3d3e2b830e81de0aa39b7af0cd6f82f759c410/src/bit_array.cr#L69). Note the empty line between the summary description and the rest.

The summary line will show up in the "Instance method summary" on top of the documentation for example.